lib/AwsEfsCreatorStackProps.ts

```diff
@@ -21,4 +21,12 @@ export interface AwsEfsCreatorStackProps extends StackProps {
      * The vpcId for the deployment.
      */
     readonly vpcId: string;
+    /**
+     * The performance mode for the EFS file system.
+     */
+    readonly performanceMode: string;
+    /**
+     * The throughput mode for the EFS file system.
+     */
+    readonly throughputMode: string;
```

Comment:
**Suggestion:** Add descriptions for the new properties `performanceMode` and `throughputMode` in the interface `AwsEfsCreatorStackProps` to specify acceptable values or examples, enhancing the clarity and usability of the API documentation. [documentation, importance: 7]
```suggestion
    /**
     * The performance mode for the EFS file system. Possible values: 'GENERAL_PURPOSE', 'MAX_IO'.
     */
    readonly performanceMode: string;
    /**
     * The throughput mode for the EFS file system. Possible values: 'BURSTING', 'PROVISIONED', 'ELASTIC'.
     */
    readonly throughputMode: string;
```